source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -369,6 +369,30 @@ progress.
 For more information on DDL operations, see
 :ref:`txn-prod-considerations-ddl`. 
 
+Network Latency
+~~~~~~~~~~~~~~~
+
+Network latency or long physical distances between migration components
+can negatively affect sync speed.
+
+Latency between ``mongosync`` and destination shards
+  For each operation on the source cluster, ``mongosync`` does two 
+  roundtrips to the destination server. The larger the latency, the 
+  slower the sync.
+
+Latency between destination shards
+  Mongosync writes operations and updates its own metadata on the 
+  destination cluster in batches in a transaction. This can result in 
+  cross-shard transactions, which may be more costly if the shards are 
+  far apart.
+
+Latency between the nodes of each individual replica set on the source or destination
+  ``mongosync`` uses :writeconcern:`"majority"` writes and 
+  :readconcern:`"majority"` reads, which require acknowledgement from 
```

Comment:
I don't think majority has to be in quotes

Reply:
Looks like we use ``"majorirty"`` pretty consistently throughout the docs, so I'm going to retain this.